modules/@apostrophecms/page/ui/apos/components/AposPagesManager.vue

```diff
@@ -306,4 +331,13 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+  .apos-pages-manager__relationship__rail {
```

Comment:
Borrowed from the pieces manager, looks good?

Reply:
pieces manager is also wrong, see `AposArrayEditor` and extend the rail component, lose the height/bg color styles